src/main/java/seedu/duke/common/SanitiseInput.java

```diff
@@ -0,0 +1,32 @@
+package seedu.duke.common;
+
+import java.util.ArrayList;
+
+public class SanitiseInput {
+    //@H-horizon
+    public static boolean sanitiseInput(String input) {
```

Comment:
Maybe the method name can be redefined to show the negative outcome it is expecting? e.g. checkForInvalidCharacters()